src/main/java/com/wenqi/bootwebdemo/dao/JFYRepo.java

```diff
@@ -0,0 +1,45 @@
+package com.wenqi.bootwebdemo.dao;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.wenqi.bootwebdemo.model.Item;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+@Repository
+public class JFYRepo {
+    private static final Logger logger = LogManager.getLogger(JFYRepo.class);
+    @Autowired
+    JdbcTemplate template;
+    private final String filePath = "target/";
```

Comment:
can be "private static final", also constant properties naming can be all caps with underscore